data-pipeline/tests/test_config.py

```
@@ -0,0 +1,94 @@
+            assert meta.get("name"), f"{code} is missing a display name"
+            assert "unit" in meta, f"{code} is missing a unit"
+
+    def test_ph_uses_range_thresholds(self):
+        ph = config.WATER_QUALITY_PARAMETERS["pH"]
+        assert ph["safe_min"] < ph["safe_max"]
+
+
+class TestIndianWaterBodies:
+    def test_coordinates_lie_within_india(self):
+        for state, meta in config.INDIAN_WATER_BODIES.items():
+            assert isinstance(meta["rivers"], list) and meta["rivers"], state
```

Ruff (0.15.21)

[warning] Assertion should be broken down into multiple parts

(PT018)